src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -147,6 +147,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('repository_url', options_list=['--context', '-c'], help="The full URL to the source code respository.")
         c.argument('commit_trigger_enabled', help="Indicates whether the source control commit trigger is enabled.", arg_type=get_three_state_flag())
         c.argument('git_access_token', help="The access token used to access the source control provider.")
+        c.argument('with-secure-properties', help="Indicates whether the secret details of builds should be returned.", action='store_true')
```

Comment:
secret details -> secure properties. Also they are not for builds, for build tasks.